### PR TITLE
Changelog for 2.5.0.1

### PR DESCRIPTION
**Pull request by lorenrh**
Signed-off-by: Lorena Rangel <lorena.rangel@docker.com>

### Proposed changes

Changelog for Docker Desktop 2.5.0.1

### CODE REVIEW

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ This page contains information about the new features, improvements, known issue
...
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
+
+### Upgrades
+
+- [Compose CLI v1.0.2](https://github.com/docker/compose-cli/releases/tag/v1.0.2)
+- [Snyk v1.424.2](https://github.com/snyk/snyk/releases/tag/v1.424.4)
+
+### Bug fixes and minor changes
+- Fix segfault in Virtual Box version check on Big Sur. See [docker/for-mac#4997](https://github.com/docker/for-mac/issues/4997).
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:27]: "Fixed an issue that caused Docker Desktop to crash on MacOS 11.0 (Big Sur) when VirtualBox was also installed."

**Review by stephen-turner — CHANGES_REQUESTED** [review]
Rewording suggested

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (usha-mandya) [inline docker-for-mac/release-notes.md:27]:
```suggestion
- Fixed segfault in Virtual Box version check on macOS Big Sur. See [docker/for-mac#4997](https://github.com/docker/for-mac/issues/4997).
```

**Comment by stephen-turner** [comment]
Oooh, @usha-mandya and I have competing suggestions. 😄

File: docker-for-mac/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ This page contains information about the new features, improvements, known issue
...
+
 ## Docker Desktop Community 2.5.0.0
 2020-11-02
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md:31]: Can we update the download URL for 2.5.0.0?

File: docker-for-windows/release-notes.md [hunk 3]
```diff
@@ -13,6 +13,16 @@ This page contains information about the new features, improvements, known issue
...
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-windows/)
+
+### Upgrades
+
+- [Compose CLI v1.0.2](https://github.com/docker/compose-cli/releases/tag/v1.0.2)
+- [Snyk v1.424.2](https://github.com/snyk/snyk/releases/tag/v1.424.4)
+
 ## Docker Desktop Community 2.5.0.0
 2020-11-02
```
Comment (usha-mandya) [inline docker-for-windows/release-notes.md:28]: Can we update the download URL for 2.5.0.0?

**Review by mat007 — COMMENTED** [review]
Should we not update the 2.5.0.0 download links to fixed URLs based including the build number?

File: docker-for-windows/release-notes.md (hunk 3 above)
Comment (mat007) [reply docker-for-windows/release-notes.md:28]: That would be https://desktop.docker.com/win/stable/49427/Docker%20Desktop%20Installer.exe

File: docker-for-mac/release-notes.md (hunk 2 above)
Comment (mat007) [reply docker-for-mac/release-notes.md:31]: This is https://desktop.docker.com/mac/stable/49427/Docker.dmg

Comment (StefanScherer) [reply docker-for-mac/release-notes.md:31]:
```
> [Download](https://desktop.docker.com/mac/stable/49427/Docker.dmg)
```

File: docker-for-windows/release-notes.md (hunk 3 above)
Comment (StefanScherer) [reply docker-for-windows/release-notes.md:28]: The download link should be

```
> [Download](https://desktop.docker.com/win/stable/49427/Docker%20Desktop%20Installer.exe)
```

**Review by StefanScherer — CHANGES_REQUESTED** [review]
Added some comments about the 2.5.0.0 download links.

File: docker-for-windows/release-notes.md (hunk 3 above)
Comment (lorenrh) [reply docker-for-windows/release-notes.md:28]: Done!

File: docker-for-mac/release-notes.md (hunk 2 above)
Comment (lorenrh) [reply docker-for-mac/release-notes.md:31]: Done!

**Review by StefanScherer — APPROVED** [review]
LGTM

**Review by stephen-turner — APPROVED** [review]
LGTM now

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**